src/pages/projectSettings/CreateProjectModal.tsx

```diff
@@ -94,18 +94,21 @@ export const CreateProjectModal: React.VFC<Props> = ({
 const modalFormDefinition = {
   schema: {
     type: "object" as "object",
+    required: ["owner", "repo"],
```

Comment:
![image](https://user-images.githubusercontent.com/13104717/182430755-8dac53a1-8b5a-4599-bec8-827d221a07b7.png)
we should add it to this modal as well